### DISCUSSION TITLE
3D model on Mobile shows only on interaction (click or scroll)

### DISCUSSION THREAD

**Discussion by Hari-byte-max** [google/model-viewer#4944] [Q&A] [answered]
I created this simple scene to test implementation of .glb model in to a WordPress website. 
[https://staffelmedien.ch/home-page-test](https://staffelmedien.ch/home-page-test) 

On the desktop model shows on page load and work as intended. On mobile, page loads (background and text), not sure if model is also loaded this time, but model will not show, until user click on the screen or scroll. 

Can anyone help me why is this happening?

**Comment by elalish** [comment]
Interesting - it appears our MV library is not loaded at all on mobile (I can repro the behavior on desktop just by opening chrome dev tools and putting the page in mobile emulation mode). I'd guess some other script on your page is deferring the loading of the MV library until interaction has occurred. If you want that for very high lighthouse scores, you should follow the example we have that does that kind of lazy loading, but with a poster image still shown: https://modelviewer.dev/examples/lighthouse.html

**Comment by Hari-byte-max** [comment, marked as answer]
Hey Emmett. Thank you for pointing me in the right direction. I check the code, and I found that Rocket WP option to prevent JS load until user interaction was ON.
Now I turned that option off and model loads immediately. :)
Problem solved


________________________________
